e2e/internal/e2e/env.go

```diff
@@ -14,3 +16,11 @@ type TestEnv struct {
 	TestRegistry  string
 	KeyringDir    string
 }
+
+// RunSingularity is a convinience wrapper for the standalone
+// RunSingularity function, ensuring that RunSingularity gets called
+// with the correct singularity path as specified by the test
+// environment.
+func (env TestEnv) RunSingularity(t *testing.T, cmdOps ...SingularityCmdOp) {
```

Comment:
I left that here as a reminder (or it's up to you if you want to tackle it in this one), to avoid confusion between different function prototypes and also for consistency with other helpers in `e2e/internal` (to pass a `TestEnv` argument instead of currently by example for a function pass a `cmdPath` and for another pass `TestEnv`), I would suggest to rename `RunSingularity(t *testing.T, cmdPath string, cmdOps ...SingularityCmdOp)` in `e2e/internal/e2e/singularitycmd.go` to `runSingularity(t *testing.T, cmdPath string, cmdOps ...SingularityCmdOp)` for internal use only by `TestEnv`, so each helpers using `RunSingularity` would take a `TestEnv` argument